### PR TITLE
Get latest configuration from comments correctly

### PR DESCRIPTION
**Pull request by kanghyojun**
Merge after https://github.com/hopsoffice/galbi/pull/2 merged.

Since GitHub issue comment API do not support sort/direction param, it has to load all comments and select latest item of payload.

### CODE REVIEW

File: galbi/cli.py [hunk 1]
```diff
@@ -140,6 +140,24 @@ def deploy(filename: str):
...
+    res = []
+    while loadable:
+        resp = fetch_func(page, per_page)
+        resp.raise_for_status()
+        payload = resp.json()
+        if per_page > len(payload):
+            loadable = False
+        res += payload
+        page += 1
+    return res
```
Comment (AiOO) [inline galbi/cli.py]: I think that it will be better to make this function return a generator with `yield`.

File: galbi/cli.py [hunk 2]
```diff
@@ -148,37 +166,29 @@ def get(key: typing.List[str]):
...
-                        'per_page': 1
+                        'per_page': per_page,
+                        'page': page,
                     }
-                ).json()[0]['body']
+                )
+            )
+            cache[d['title']] = json.loads(
+                comments[-1]['body']
             )
```
Comment (AiOO) [inline galbi/cli.py]: Why don't for-loop `break`s after found all desired issues?

**Review by AiOO — COMMENTED** [review]
If the issue has duplicated key—_title, label_— is not permitted, we can prevent redundant API calls.

File: galbi/cli.py (hunk 2 above)
Comment (AiOO) [reply galbi/cli.py:192]: Oh there is `keys` not `key` 🤦

**Comment by kanghyojun** [comment]
@AiOO `fetch_all_pages` has two use-cases.
 
1. To get issues match with `key`: Since `keys` has multiple values in it, If match every `key` in for loop, we can do `break` iin it.
2. To get a latest comment of issue: We has to get all comments to get latest one, There is no benefit to use iterator over list.

It has few benefit if we had a iterator with specific case. But, still generator is more efficient than list, I modified return type of `fetch_all_pages` to iterator.

**Review by AiOO — APPROVED** [review]
LGTM. Thanks for applying my review.

**Pull request merged**